Source/santa/SNTNotificationManager.m

```diff
@@ -207,36 +208,52 @@ - (void)hashBundleBinariesForEvent:(SNTStoredEvent *)event
 #pragma mark SNTNotifierXPC protocol methods
 
 - (void)postClientModeNotification:(SNTClientMode)clientmode {
-  NSUserNotification *un = [[NSUserNotification alloc] init];
-  un.title = @"Santa";
-  un.hasActionButton = NO;
-  NSString *customMsg;
+  UNUserNotificationCenter *un = [UNUserNotificationCenter currentNotificationCenter];
+
+  UNMutableNotificationContent *content = [[UNMutableNotificationContent alloc] init];
+  content.title = @"Santa";
+
   switch (clientmode) {
-    case SNTClientModeMonitor:
-      un.informativeText = @"Switching into Monitor mode";
-      customMsg = [[SNTConfigurator configurator] modeNotificationMonitor];
-      if (!customMsg) break;
-      if (!customMsg.length) return;
```

Comment:
> if (!customMsg.length) return;

This seems like a bug and I think it's right to fix it the way you did. But just to be sure, do you think anyone is replying on this behavior? If someone puts an empty string in the custom message config they won't get a popup. Seems like a bug.